packages/cli/src/pull/handler.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  DeployConfig,
+  getProject,
+  getConfig,
+  getState ,
+} from '@openfn/deploy';
+import type { Logger } from '../util/logger';
+import { DeployOptions } from '../deploy/command';
+import path from 'path';
+import fs from 'node:fs/promises';
+
+
+
+
+async function pullHandler(
+  options: DeployOptions,
+  logger: Logger,
+) {
+  try {
+    const config = mergeOverrides(await getConfig(options.configPath), options);
+    logger.always("Downloading project yaml and  state from instance");
+    const state = await getState(config.statePath);
```

Comment:
The only thing we use state for here is  to get the project id. Do we really need to do that? Is it not available anywhere else?

Reply:
I am likely going to change this so it comes in as command line param